OpenRA.Mods.Common/Traits/Cargo.cs

```diff
@@ -142,6 +142,19 @@ void INotifyCreated.Created(Actor self)
 		{
 			aircraft = self.TraitOrDefault<Aircraft>();
 			conditionManager = self.TraitOrDefault<ConditionManager>();
+
+			if (Info.InitialUnits.Any())
+			{
+				foreach (var u in Info.InitialUnits)
```

Comment:
Shoulda use the actual cargo itself as the source of iteration instead of taking this from info, because you can initialize individual actors with specifying their cargo on the map, which might not overlap with what the info says - and you already have the actors initialized themselves by this point.